network/NetworkGen/templates/MessageUnpack_generated.h.mako

```diff
@@ -33,6 +33,13 @@
 %for node in nodes:
   %for bus in node.on_buses:
 
+extern CANRX_${bus.upper()}_signals_S CANRX_${bus.upper()}_signals;
```

Comment:
imo you should just generate this file multiple times, once per bus.
Something like `MessageUnpack_VEH_generated.h`
Then the loops get simpler and the file gets more readable I think

Reply:
I ended up doing it once per component so that you minimize the ram footprint. An ecu only cares about the signals its rxing so to save space went with it like this